### PR TITLE
reporter

### PR DESCRIPTION
**Pull request by Christianlopez4**
Captura

### CODE REVIEW

File: protractor/local.config.ts [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import { Config, browser } from "protractor";
+import { reporter } from "./helpers/reporter";
 
 export const config: Config = {
   framework: "jasmine",
   specs: ["../test/google.spec.js"],
   seleniumAddress: "http://localhost:4444/wd/hub",
   onPrepare: () => {
     browser.ignoreSynchronization = true;
+    reporter();
```
Comment (holgiosalos) [inline protractor/local.config.ts:10]: reporter should go outside of the onPrepare function

File: package.json [hunk 2]
```diff
@@ -25,6 +25,7 @@
   },
   "homepage": "https://github.com/Christianlopez4/protractor-workshop-20201#readme",
   "dependencies": {
+    "jasmine-spec-reporter": "^5.0.2",
```
Comment (holgiosalos) [inline package.json]: please install jasmine and protractor as dev dependencies.

File: protractor/local.config.ts (hunk 1 above)
Comment (Christianlopez4) [reply protractor/local.config.ts:10]: When i put the reporter outside of the onPrepare function an error is generated

File: package.json (hunk 2 above)
Comment (Christianlopez4) [reply package.json]: i already installed jasmine as dev dependency, but when i tried to do the same with protractor an error was generated
Captura

File: protractor/local.config.ts (hunk 1 above)
Comment (holgiosalos) [reply protractor/local.config.ts:10]: yes, you're right, I'm sorry, got confused.

File: package.json (hunk 2 above)
Comment (holgiosalos) [reply package.json]: did you run `npm install --save-dev protractor` command?

Comment (Christianlopez4) [reply package.json]: yes, i did it

Comment (holgiosalos) [reply package.json]: run `npm ci` to reinstall all dependencies.

Comment (Christianlopez4) [reply package.json]: This is strange, the error continues there, but I was able to run the npm test without issue

Comment (Christianlopez4) [reply package.json]: i'll upload it for you to collaborate with me

**Review by holgiosalos — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (holgiosalos) [reply package.json]: I cloned your repo, ran `npm ci` then `npm run test`, also opened with VSCode and didn't get any error, so I guess that everything is fine.

**Pull request merged**